src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -29,11 +36,32 @@ def get_attrs(self, item_list, user, **kwargs):
 
         result = defaultdict(dict)
         triggers = AlertRuleTrigger.objects.filter(alert_rule__in=item_list).order_by("label")
-        serialized_triggers = serialize(list(triggers))
+        serialized_triggers = serialize(list(triggers), **kwargs)
+
+        trigger_actions = AlertRuleTriggerAction.objects.filter(
+            alert_rule_trigger__alert_rule_id__in=alert_rules.keys()
+        ).exclude(sentry_app_config__isnull=True, sentry_app_id__isnull=True)
+
+        sentry_app_installations_by_sentry_app_id = (
+            SentryAppInstallation.objects.get_related_sentry_app_components(
+                organization_id__in={
```

Comment:
This is not a `filter()` call. You'll need to pass `organization_ids` as a parameter.

Reply:
good catch!